applications/Bright_Disputes.md

```diff
@@ -0,0 +1,180 @@
+
+![](https://i.imgur.com/WjZczZx.png)
+
+Our target plaform for the deployment will be AlephZero, which provides zk-snarks verification.
+
+This project will not provide any UI. All the functionality can be tested by the polkadot.js dApp.
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Michał Graliński - Rust Developer
+- Katarzyna Łukasiewicz - Project Manager
+
+### Contact
+
+* **Contact Name:** Katarzyna Łukasiewicz
+* **Contact Email:** katarzyna.lukasiewicz@brightinventions.pl
+* **Website** https://brightinventions.pl/
+
+### Legal Structure
+
+- **Registered Address:** ul. Jana Matejki 12, 80-232 Gdańsk, Poland
+- **Registered Legal Entity:** Bright Inventions Sp. z o. o.
+
+info@brightinventions.pl
+
+www.brightinventions.pl
+
+
+Bright Inventions is a team of oveer 70 full-time onsite developers, project managers & UX/UI designers - experts in mobile and web applications, systems integration, IOT devices and Blockchain platforms. 
+
+
+We believe that building a software product is about people working together in a collective way. By offering complex support – mobile and web development as well as IT consultancy we try to eliminate roadblocks towards engaging clients as partners at every step of the process. 
+
+
+We support startups, digital agencies as well as medium to big businesses. We cooperate with startups, accelerators and incubators. Whatever the client profile is, we always  aim to establish a satisfying partnership for both sides. Since 2012 we have built software for more than 40 businesses worldwide.
+
+### Team's experience
+As a company we have a 4 year experience with blockchain technologies, including cooperation with Parity. Since almost 2 years we’re developing our own Substrate based product [BrightTreasury](https://treasury.bright.dev/), that's been funded first by Web3 Foundation Grants program and has been supported by Polkadot and Kusama treasuries since. More recently, we’ve engaged in Rust development for Substrate as well, with two Web3 Foundation grants: [ZK-Snarks tutorial](https://github.com/w3f/Grants-Program/pull/1290) and [High-availability validator setup with Raft](https://github.com/w3f/Grant-Milestone-Delivery/pull/813).
+
+Who would work on Bright Disputes?
+- Michał - Rust Developer, for over 12 years he has been working as a C++ developer. In his career he participate in various projects, starting from the embedded devices, automotive, mobile games and ending on the financial systems. Right now he is focused on the blockchain technology.
+- Kasia - she has been working with agile methods for over 10 years, both as a researcher and a practitioner. She believes in teamwork and a power of user centered mindset. With experience in leading international projects, she knows how to support and encourage timely and high quality deliveries.
+
+### Team Code Repos
```

Comment:
I don't really see any code activity from either team member. Do you have any other code samples or references you can provide?

Reply:
Indeed, we usually work under our Bright Inventions name, hence the https://github.com/bright is the repository that contains the code (it's the one mentioned above our github links). Some of the Rust work can be found for example here: https://github.com/bright/zk-snarks-with-substrate/ and here https://github.com/bright/substrate-raft/